tests/ecs6-class/line.test.js

```diff
@@ -6,17 +6,21 @@ const point1 = new Point({ x: 4, y: 3 });
 const point2 = new Point({ x: 3, y: 2 });
 const line = new Line({ point1, point2 });
 const line1 = new Line({});
+line1.slope = 0;
+const line2 = new Line({ point1, point2 });
+line1.n = 0;
+const line3 = new Line({});
+line3.slope = 2;
+const line4 = new Line({ point1, point2 });
+line4.n = 0;
```

Comment:
Don't define some objects in the begin of the module, each test should get its own objects,